index.js

```diff
@@ -62,22 +76,27 @@ conventionalRecommendedBump({
     console.error(chalk.red(err.message))
     return
   }
-
   var newVersion = pkg.version
   if (!argv.firstRelease) {
-    newVersion = semver.inc(pkg.version, release.releaseAs)
+    newVersion = getNextVersion(release)
     checkpoint('bumping version in package.json from %s to %s', [pkg.version, newVersion])
     pkg.version = newVersion
     fs.writeFileSync(pkgPath, JSON.stringify(pkg, null, 2) + '\n', 'utf-8')
   } else {
     checkpoint('skip version bump on first release', [], chalk.red(figures.cross))
   }
-
-  outputChangelog(argv, function () {
+  // only commit the package.json on prerelease
```

Comment:
I'm with @bcoe, the end goal should be combining them to a single entry eventually. Another option is to collect the changes in a `Unrelease vX.Y.Z` and then name it to `vX.Y.Z` once it's not a pre-release anymore. I could swear I've seen that around, although I'm too lazy now to search for it.